python_on_whales/utils.py

```diff
@@ -219,8 +219,12 @@ def stream_stdout_and_stderr(
     process = Popen(full_cmd, stdout=PIPE, stderr=PIPE, env=subprocess_env)
     q = Queue()
     full_stderr = b""  # for the error message
-    Thread(target=reader, args=[process.stdout, "stdout", q]).start()
-    Thread(target=reader, args=[process.stderr, "stderr", q]).start()
+    th = Thread(target=reader, args=[process.stdout, "stdout", q])
```

Comment:
```suggestion
    # we use deamon threads to avoid hanging if the user uses ctrl+c
    th = Thread(target=reader, args=[process.stdout, "stdout", q])
```